### PR TITLE
14.0 mig website product configurator mrp

### PR DESCRIPTION
**Pull request by pledrateam**

### CODE REVIEW

**Comment by PCatinean** [comment]
MR ready for review

**Review by PCatinean — APPROVED** [review]

File: website_product_configurator_mrp/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (patrickrwilson) [inline website_product_configurator_mrp/__manifest__.py:2]: Please add licensing info.

File: website_product_configurator_mrp/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+from odoo import http
```
Comment (patrickrwilson) [inline website_product_configurator_mrp/controllers/main.py:2]: Please add licensing info.

File: website_product_configurator_mrp/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#Odoo Website Product Configurator MRP
```
Comment (patrickrwilson) [inline website_product_configurator_mrp/README.md]: Please make the formatting similar to other OCA readme standards.

File: website_product_configurator_mrp/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+from . import controllers
```
Comment (patrickrwilson) [inline website_product_configurator_mrp/__init__.py:2]: Please add licensing info.

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
Code review only, just needs licensing info

**Review by patrickrwilson — APPROVED** [review]
LGTM

**Pull request merged**